packages/electron-publish/src/gitHubPublisher.ts

```diff
@@ -215,6 +215,19 @@ export class GitHubPublisher extends HttpPublisher {
       })
   }
 
+  private doesErrorMeanAlreadyExists(e: any) {
+    let doesErrorMeanAlreadyExists = true
+    doesErrorMeanAlreadyExists &&= e.statusCode === 422
```

Comment:
this ends up transpiling to `doesErrorMeanAlreadyExists && (doesErrorMeanAlreadyExists = e.statusCode === 422);` which seems wrong to me, perhaps I need not use `&&=` which now that I read the [moz spec](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND_assignment) is not meant for assignment ... for whatever strange reason.

Reply:
pushed an update